test/parallel/test-datetime-change-notify.js

```diff
@@ -26,7 +26,7 @@ const cases = [
   },
   {
     timeZone: 'Europe/Dublin',
-    expected: /Irish/,
+    expected: /(Irish Standard Time|Greenwich Mean Time)/,
```

Comment:
It's probably best to avoid unnecessary groups:

```suggestion
    expected: /Irish Standard Time|Greenwich Mean Time/,
```